SDK/Implement/MasterWalletManager.cpp

```diff
@@ -294,37 +340,57 @@ namespace Elastos {
 		MasterWalletManager::ExportWalletWithKeystore(IMasterWallet *masterWallet, const std::string &backupPassword,
 													  const std::string &payPassword) const {
 
+			Log::preinfo("{} | {} | * | * ", __FUNCTION__, masterWallet->GetBasicInfo().dump());
+
 			ErrorChecker::CheckParam(masterWallet == nullptr, Error::InvalidArgument, "Master wallet is null");
 			ErrorChecker::CheckPassword(backupPassword, "Backup");
 
 			MasterWallet *wallet = static_cast<MasterWallet *>(masterWallet);
-			return wallet->exportKeyStore(backupPassword, payPassword);
+			nlohmann::json keystore = wallet->exportKeyStore(backupPassword, payPassword);
+			Log::retinfo("{} | {}", __FUNCTION__, keystore.dump());
+			return keystore;
 		}
 
 		std::string
 		MasterWalletManager::ExportWalletWithMnemonic(IMasterWallet *masterWallet, const std::string &payPassword) const {
 
+			Log::preinfo("{} | * | * ", __FUNCTION__);
```

Comment:
This will be better -> Log::preinfo("{} | {} | *", __FUNCTION__, masterWallet->GetBasicInfo().dump());